iam_ltk.go

```diff
@@ -21,6 +21,12 @@ type GetLongTermKeysResponse struct {
 	LongTermKeys []LongTermKey `json:"longTermKeys"`
 }
 
+// GetLongTermKeyResponse is used to represent a single long term key.
+type GetLongTermKeyResponse struct {
+	BaseResponse
+	LongTermKey `json:"longTermKey"`
```

Comment:
Careful.  This is called `embedding` and actually makes your `GetLongTermKeyResponse` a subclass of both BaseResponse (which is true) and `LongTermKey` which is kinda weird?  You probably want this to be a named field unless your intent is to have users see this response as having the bare long term key fields available AND you reuse the collection of those fields in other places in the codebase as a unit.  What I am trying to get at is that this is not very idiomatic `golang`, so we need to have a good reason for it to be that way or follow the more traditional approach of naming this field (probably also named `LongTermKey`).

Reply:
Reading more of the file, maybe this is OK as is.  Leaving the note so we can discuss.